mlrun/runtimes/base.py

```diff
@@ -399,8 +399,8 @@ def _get_db_run(self, task: RunObject = None):
 
     def _get_cmd_args(self, runobj, with_mlrun):
         extra_env = {'MLRUN_EXEC_CONFIG': runobj.to_json()}
-        if self.spec.rundb:
-            extra_env['MLRUN_DBPATH'] = self.spec.rundb or config.dbpath
+        # if self.spec.rundb:
+        #     extra_env['MLRUN_DBPATH'] = self.spec.rundb or config.dbpath
```

Comment:
@Hedingber in-cluster tasks should use the in-cluster service path, w/o this change it will try to use the external service url and will fail auth (since we dont give the pod the platform password env)